pandas/core/series.py

```diff
@@ -2878,16 +2878,22 @@ def from_csv(cls, path, sep=',', parse_dates=True, header=None,
 
         return result
 
-    def to_csv(self, path=None, index=True, sep=",", na_rep='',
+    @deprecate_kwarg(old_arg_name='path', new_arg_name='path_or_buf')
+    def to_csv(self, path_or_buf=None, index=True, sep=",", na_rep='',
```

Comment:
ideally we should have a shared doc-string defined in generic.py (parmeterized by klass) to avoid repeating all of it.

Reply:
Is there a best-practices example for this that I could use for guidance?

Reply:
there are lots of examples virtually every function in generic, e.g. ``.reindex``

Reply:
I think it will be easier to wait with moving to generic.py until deprecation of positional args is done

Reply:
deprecating positional parameters is pretty silly here and will be way too messy. just conform the signature, write a nice message in the whatsnew and let's call it a day.

Reply:
It is perfectly possible to do this (deprecating positional arguments), so why not do that? IMO there is no reason in possibly breaking user's code if we can easily prevent it.

Reply:
go for it then. this is going to be a big mess.